source/utilities.tex

```diff
@@ -40,7 +40,7 @@
 \indextext{\idxhdr{utility}}%
 \indexlibrary{\idxhdr{utility}}%
 \indexlibrary{\idxcode{rel_ops}}%
-\synopsis{Header \tcode{<utility>} synopsis}
+\rSec2[utility.syn]{Header \tcode{<utility>} synopsis}
```

Comment:
Right, with this change we would have a numbered section called "Synopsis" that does *not* start with a code block. That'd be new, and moreover it seems that all these introductory sentences don't really add any normative value. So maybe we could revise this on at a slightly larger scale?